x-pack/solutions/observability/plugins/observability/server/lib/rules/custom_threshold/utils.ts

```diff
@@ -212,6 +212,26 @@ export const flattenObject = (obj: AdditionalContext, prefix: string = ''): Addi
     return acc;
   }, {});
 
+export const getGroupByObject = (
```

Comment:
Shall we add a test for this?

Reply:
added test in acdddd148ce1a551a32caeb6f6b178b31ace08f5